### PR TITLE
OCM-6516 | fix: Improved help text for config get/set commands

### PR DESCRIPTION
**Pull request by tirthct**
**Changed**
- Improved help text for config get/set commands

https://issues.redhat.com/browse/OCM-6516

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -46,6 +48,31 @@ type Config struct {
 	FedRAMP      bool     `json:"fedramp,omitempty" doc:"Indicates FedRAMP."`
 }
 
+var DisallowedSetConfigProperties = []string{"scopes"}
+
+func ConfigPropertiesNamesAndDocs() ([]string, []string) {
```
Comment (robpblake) [inline pkg/config/config.go:53]: @tirthct Could we add a unit test for this please

File: cmd/config/get/get.go [hunk 2]
```diff
@@ -37,12 +38,18 @@ func NewConfigGetCommand() *cobra.Command {
...
-		Args:  cobra.ExactArgs(1),
-		Run:   run,
+		Long: fmt.Sprintf("Prints the value of a config variable. Supported variables are: %s.",
+			strings.Join(GetAllConfigProperties(), ", ")),
+		Args: cobra.ExactArgs(1),
+		Run:  run,
 	}
 }
 
+func GetAllConfigProperties() []string {
```
Comment (robpblake) [inline cmd/config/get/get.go:48]: @tirthct Shall we move this into `config.go` so it's closer to the `ConfigPropertiesNamesAndDocs()` function and the pattern remains that the `command` implementations simply call a function and do minimal logic themselves?

File: cmd/config/set/set.go [hunk 3]
```diff
@@ -33,10 +36,22 @@ func NewConfigSetCommand() *cobra.Command {
...
-		Args:  cobra.ExactArgs(2),
-		Run:   run,
+		Long: fmt.Sprintf("Sets the value of a config variable. Supported variables are: %s.",
+			strings.Join(getAllowedConfigProperties(), ", ")),
+		Args: cobra.ExactArgs(2),
+		Run:  run,
+	}
+}
+
+func getAllowedConfigProperties() []string {
```
Comment (robpblake) [inline cmd/config/set/set.go:46]: @tirthct Same comment here about moving this into `config.go`. And also a unit test.

File: cmd/config/get/get.go [hunk 4]
```diff
@@ -37,12 +38,18 @@ func NewConfigGetCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "get [flags] VARIABLE",
 		Short: "Prints the value of a config variable",
-		Long:  "Prints the value of a config variable. See 'rosa config --help' for supported config variables.",
-		Args:  cobra.ExactArgs(1),
-		Run:   run,
+		Long: fmt.Sprintf("Prints the value of a config variable. Supported variables are: %s.",
+			strings.Join(GetAllConfigProperties(), ", ")),
```
Comment (robpblake) [inline cmd/config/get/get.go:42]: @tirthct The general pattern seems to be to print supported values on a new line e.g `rosa help config`

File: pkg/config/config.go (hunk 1 above)
Comment (tirthct) [reply pkg/config/config.go:55]: Added tests

File: cmd/config/get/get.go (hunk 2 above)
Comment (tirthct) [reply cmd/config/get/get.go]: Makes sense. Moved

File: cmd/config/set/set.go (hunk 3 above)
Comment (tirthct) [reply cmd/config/set/set.go]: Moved.

File: cmd/config/get/get.go (hunk 4 above)
Comment (tirthct) [reply cmd/config/get/get.go]: Changed to print in new line

**Review by robpblake — APPROVED** [review]
/lgtm

**Pull request merged**